presto-main/src/main/resources/webapp/src/package.json

```diff
@@ -12,17 +12,22 @@
     "@babel/preset-flow": "^7.22.15",
     "@babel/preset-react": "^7.22.15",
     "babel-loader": "9.1.3",
+    "css-loader": "^6.8.1",
     "flow-bin": "^0.215.1",
+    "style-loader": "^3.3.3",
     "webpack": "5.88.2",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^4.15.1"
   },
   "dependencies": {
+    "@prestodb/presto-js-client": "^0.1.0-beta.0",
```

Comment:
Latest version is should be [presto-client-0.1.1](https://github.com/prestodb/presto-js-client/releases/tag/presto-client-0.1.1)

Reply:
thanks @jorgeramirezamora 
fixed. I didn't specify the version when I ran `yarn add @prestodb/presto-js-client` and the interesting thing is the version shows up in [here](https://www.npmjs.com/package/@prestodb/presto-js-client?activeTab=readme) is not 0.1.1. But 0.1.1 is the latest version indeed. I will leave that to you to handle it.

Reply:
Yeah, GitHub is marking `0.1.1` as LTS but NPM is marking `0.1.0-beta.0` as LTS. Will bump it to `0.2.0` in the coming days to avoid this issue.